docs/source/introduction.md

```diff
@@ -177,50 +177,57 @@ with the bbox.
 
 Note: the `bbox` values must be in the same CRS as the dataset.
 
+Note: if GEOS is present and used by GDAL, only geometries that intersect `bbox`
```

Comment:
This is the only substantive change in this file, everything else is auto-formatter stripping of trailing whitespace at end of lines.  Sorry for the noise here.